### PR TITLE
feat: conformance for /did.json and /identifiers/:did

### PR DESCRIPTION
**Pull request by brownoxford**
Add conformance testing for the `/did.json` and `/identifiers/:did` endpoints.

REF: #252 
REF: #251

### CODE REVIEW

File: tests/conformance_suite.postman_collection.json [hunk 1]
```diff
@@ -1,11 +1,264 @@
...
+						}
+					]
+				}
+			]
+		},
+		{
+			"name": "Resolve Identifiers",
+			"item": [
+				{
+					"name": "Negative Testing",
```
Comment (OR13) [inline tests/conformance_suite.postman_collection.json:59]: nice

File: tests/conformance_suite.postman_collection.json [hunk 2]
```diff
@@ -2777,6 +3030,16 @@
 			"key": "responseSchema500",
 			"value": "{\"type\":\"object\",\"required\":[\"code\",\"message\"],\"properties\":{\"code\":{\"type\":\"integer\"},\"message\":{\"type\":\"string\"},\"details\":{\"type\":\"object\"}}}",
 			"type": "string"
+		},
+		{
+			"key": "responseSchema200Identifiers",
```
[the remark this answers is not in the document]
Comment (brownoxford) [reply tests/conformance_suite.postman_collection.json:3035]: Would love another option as well, do you want to create an issue to investigate options?

[the remark this answers is not in the document]
Comment (OR13) [reply tests/conformance_suite.postman_collection.json:3035]: I don't know if there are any, imo its a nice to have... no need to track in a ticket... yet.

File: tests/conformance_suite.postman_collection.json [hunk 3]
```diff
@@ -1,11 +1,264 @@
...
 		"name": "Conformance Suite",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
 		"_exporter_id": "4338127"
 	},
 	"item": [
+		{
+			"name": "API Configuration",
+			"item": [
+				{
+					"name": "Happy Path",
```
[the remark this answers is not in the document]
Comment (brownoxford) [reply tests/conformance_suite.postman_collection.json:13]: Almost certainly #291

**Review by BenjaminMoe — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

File: tests/conformance_suite.postman_collection.json [hunk 4]
```diff
@@ -6,6 +6,259 @@
...
```
Comment (TallTed) [inline tests/conformance_suite.postman_collection.json:13]: Maybe?
```suggestion
					"name": "Positive Testing",
```

**Review by nissimsan — APPROVED** [review]

**Pull request merged**